packages/tasks_panel/src/misc/e2e-config.ts

```diff
@@ -0,0 +1,220 @@
+import { GlobPattern, TaskDefinition, Uri, extensions, workspace } from "vscode";
+import { BasToolkit } from "@sap-devx/app-studio-toolkit-types";
+import {
+  Dictionary,
+  compact,
+  concat,
+  extend,
+  find,
+  includes,
+  isEmpty,
+  isMatch,
+  last,
+  reduce,
+  size,
+  split,
+} from "lodash";
+import { getUniqueTaskLabel, updateTasksConfiguration } from "../../src/utils/task-serializer";
+import { DEFAULT_TARGET, cfGetConfigFileField, cfGetTargets } from "@sap/cf-tools";
+import { getLogger } from "../../src/logger/logger-wrapper";
+import * as path from "path";
+
+export enum ProjectTypes {
```

Comment:
don't use enums in TypeScript
https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh